dipy/reconst/sfm.py

```diff
@@ -18,21 +18,25 @@
 import warnings
 
 import numpy as np
+import gc
 
 try:
     from numpy import nanmean
 except ImportError:
     from scipy.stats import nanmean
 
 from dipy.utils.optpkg import optional_package
+from dipy.utils.multiproc import determine_num_processes
 import dipy.core.gradients as grad
 import dipy.core.optimize as opt
 import dipy.sims.voxel as sims
 import dipy.data as dpd
 from dipy.reconst.base import ReconstModel, ReconstFit
 from dipy.reconst.cache import Cache
 from dipy.core.onetime import auto_attr
+from collections import OrderedDict
```

Comment:
Can you put the import of `gc` and `collections` with `warnings`. We try to classiffy them.
1. standard library
2. third party library
3. local import

Reply:
Fixed!